### ISSUE TITLE
Physical unit of final depth map

### ISSUE THREAD

**Issue by t-fi** [lmb-freiburg/demon#34]
What is the physical unit of the final depth map? Is there a easy way to obtain a depth map e.g. in meters?

I guess one has to first invert the inverse depth back to the physical depth. But then it still needs to be converted into SI-units. I tried to eyeball the conversion factor, but could not find a consistent number.

I could imagine that the scale factor (the s from in the paper) cannot be estimated very accurately. Just imagine having two image pairs: one of a usual lego figure, and one of a human sized one. There would be no way to tell them apart right?

**Comment by benjaminum** [comment]
Yes, in general it is not possible to infer the scale.

The predicted depth map is scaled such that the depth values correspond to the unit translation.
If you know that the translation is about 1m then the depth values are also roughly in meters.

**Comment by t-fi** [comment]
Thanks for the confirmation. While skimming the paper I stumbled upon the fact that the 'rescaling' of the depthmap actually happens after the iterative but before the refinement network. From the top of my mind it would seem more natural to rescale after passing the data through all the networks. 

Is this maybe required for training somehow, or am I maybe missing something else?

Also if you would rather use github issues for implementation discussion only, I could contact you via email.

**Comment by benjaminum** [comment]
Yes, this is because of the training procedure, which uses multiple stages.

Using github issues for these questions is fine.
